hack/verify-versionize-docs.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verifies that running versionize-docs the second time, does not result in any
+# diff.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+API_ROOT="${KUBE_ROOT}/pkg/api/"
+SWAGGER_SPEC_ROOT="${KUBE_ROOT}/api/swagger-spec/"
+DOCS_ROOT="${KUBE_ROOT}/docs/"
+EXAMPLES_ROOT="${KUBE_ROOT}/examples/"
+TMP_ORIGINAL_ROOT="/tmp/tmp-original-files/"
+TMP_VERSIONED_ROOT="/tmp/tmp-versioned-files/"
+
+# Copy the original version of all files that are going to be changed by
+# versionize-docs.
+echo "Copying original docs to ${TMP_ORIGINAL_ROOT}"
+mkdir -p "${TMP_ORIGINAL_ROOT}"
+cp -a "${DOCS_ROOT}" "${TMP_ORIGINAL_ROOT}"
+cp -a "${EXAMPLES_ROOT}" "${TMP_ORIGINAL_ROOT}"
+mkdir "${TMP_ORIGINAL_ROOT}pkg"
+cp -a "${API_ROOT}" "${TMP_ORIGINAL_ROOT}pkg"
+mkdir "${TMP_ORIGINAL_ROOT}api"
+cp -a "${SWAGGER_SPEC_ROOT}" "${TMP_ORIGINAL_ROOT}api"
+
+# Run versionize-docs.
+echo "Running versionize-docs for the first time"
+${KUBE_ROOT}/build/versionize-docs.sh v1 >/dev/null
```

Comment:
Having some sort of test for this script is probably good.

But it seems kind of bad to have the test mutate the repo itself.

My suggestion is to make a copy, run versionize on the copy, and then run `mungedocs --verify` on the copy. My original (poorly worded!) complaint was that I had to run mungedocs after running versionize to get it to settle down. Sorry for the confusion!